server.js

```diff
@@ -985,6 +986,36 @@ app.post('/submit', (req, res) => {
     }
 });
 
+app.get('/rss', asyncMiddleware(async (req, res, next) => {
+    var rss_path = path.join(__dirname, 'static', 'rss.xml')
+      , best_network_path = path.join(__dirname, 'network', 'best-network.gz')
+      , should_generate = true;
+    
+    var rss_exists = await fs.pathExists(rss_path)
+        
+    if(rss_exists) {
+        best_network_mtimeMs = (await fs.stat(best_network_path)).mtimeMs;
```

Comment:
Each time the directory changes it'll also change the mtimeMs of the best network, even if we don't have a new best network. So then we'll need to re-calc the best network hash each time this changes here from an rss request? Should it be a local variation instead and not change/use best_network_mtimeMs at all?